### ISSUE TITLE
`-c 1` interestingly leads to different error than `-c 5`

### ISSUE THREAD

**Issue by kjappelbaum** [texworld/betterbib#275]
Perhaps just a curiosity, but it is a bit unintuitive 

```
betterbib sync references.bib --in-place -c 1
Warning: Don't know how to parse name `Shoemaker, Benjamin A and Thiessen, Paul A and Yu, Bo
Syncing... ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   0% -:--:--
Warning: Not Found! (404)
Try again later.
Synced 0 of 44 entries
```

gives a different error than 

```
betterbib sync references.bib --in-place
Warning: Don't know how to parse name `Shoemaker, Benjamin A and Thiessen, Paul A and Yu, Bo
Syncing... ━━╸━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━   7% -:--:--
Warning: Too Many Requests! (429)
Try again later.
Synced 3 of 44 entries
```

almost suggesting that either `c=1` is not allowed or a different route is queried.

**Comment by kjappelbaum** [comment]
in general, I would also expect a library to handle the load balancing for me and to not ask me to try again later. (Again, perhaps this happens already internally but at least the errors suggest this does not happen)

**Comment by nschloe** [comment]
Fixed in 7.1.3.
